api/models/Post.js

```diff
@@ -253,6 +253,11 @@ module.exports = bookshelf.Model.extend(Object.assign({
         }).save().then(inc(1)))
     }))
     .then(() => this)
+  },
+
+  removeFromCommunity: function (slug) {
+    return PostMembership.find(this.id, slug)
```

Comment:
can you name this idOrSlug since it works either way? otherwise this is good to merge